index.html

```diff
@@ -264,12 +264,17 @@ <h2 class="form-heading">Contact me</h2>
             </label>
 
             <label for="your email">
-              <input type="email" id="email-id" placeholder="example@gmail.com" name="email-id" required />
+              <input type="email" id="email-id" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$"
+              placeholder="example@gmail.com" name="email-id" required />
```

Comment:
- [ ] Kindly use a custom error message. Currently, it's vague and doesn't inform the user much about the "requested format".

## Current
![image](https://user-images.githubusercontent.com/92215878/202477861-f4e3ebf2-3eb2-4b9d-b78f-fb9b75ef3c06.png)

## Requested
![image](https://user-images.githubusercontent.com/92215878/202478216-f1bc815a-48e9-4900-82ae-9d30accfafe9.png)

_You can create a function that checks the input from the user and renders an error message if the format isn't correct 😄_